6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py

```diff
@@ -200,6 +194,43 @@
 # In[11]:
 
 
+list_of_enrichment_dfs = []
+for data_shuffle_df in output_dfs:
+    df = output_dfs[data_shuffle_df]
+    # split the feature column
+    df[
+        [
+            "feature_group",
+            "measurement",
+            "bone",
+            "parameter1",
+            "parameter2",
+            "parameter3",
```

Comment:
Are these parameters defined somewhere? Currently, the naming is very vague. Note, I am coming from a very minimal background of the project so maybe this was already defined somewhere else.

Reply:
Are these CellProfiler features maybe?

Reply:
Yes, and I do not use the parameters, they are just place holders